iceoryx_posh/test/moduletests/test_roudi_portpool.cpp

```diff
@@ -391,6 +452,226 @@ TEST_F(PortPool_test, RemoveSubscriberPortIsSuccessful)
     EXPECT_EQ(subscriberPortDataList.size(), 0U);
 }
 
+// END SubscriberPort tests
+
+// BEGIN ClientPort tests
+
+TEST_F(PortPool_test, AddClientPortIsSuccessful)
```

Comment:
I'm not that happy with the naming but all tests in this file follow the same schema and this makes it easier to check if all ports have the same tests